spec/requests/profile_field_groups_request_spec.rb

```diff
@@ -1,15 +1,19 @@
 require "rails_helper"
 
 RSpec.describe "ProfileFieldGroups", type: :request do
-  describe "GET /api/profile_field_groups" do
-
-    before { sign_in(create(:user)) }
+  let(:user) { create(:user) }
 
+  describe "GET /api/profile_field_groups" do
```

Comment:
Since we're here already, could you please remove the `api/` part, as it's not actually true.